src/sql/upgrade-42.sql

```diff
@@ -0,0 +1 @@
+alter table Builds add column isChannel integer not null default 0;
```

Comment:
As for the second comment, I think that's better to do in a different branch/pull request, because we have quite a lot of integer columns that are used as booleans.